Cargo.lock

```diff
@@ -365,9 +365,9 @@ dependencies = [
 
 [[package]]
 name = "regex-syntax"
-version = "0.6.25"
+version = "0.6.27"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "f497285884f3fcff424ffc933e56d7cbca511def0c9831a7f9b5f6153e3cc89b"
+checksum = "a3f87b73ce11b1619a3c6332f45341e0047173771e8b8b73f87bfeefb7b56244"
```

Comment:
Are these upgrades of `regex` and `regex-syntax` relevant to the UDP bind fix? Otherwise I think we should remove them from this PR. If we need to upgrade dependencies we can do so in a separate PR. But I see no reason to do that unless we want to build a release, because the lockfile is not used when consumed as a library.